modules/roles_profiles/manifests/roles/gecko_t_linux_talos.pp

```diff
@@ -18,6 +18,15 @@
     #include ::fw::roles::linux_taskcluster_worker
 
     # talos stuff
-    include ::roles_profiles::profiles::gecko_t_linux_talos_generic_worker
-
+    case $::operatingsystemrelease {
```

Comment:
Do we need to change this / have an if block? Aren't we specifying the 2204 profile in the 2204 role profile?

Reply:
They both point at this shared file. I might as well just make them its own files so 18.04 is easier to clean up

Reply:
This was put there because I thought something was broken. Turns out that the kitchen config was wrong so it never used the 2204 one. This change has been reverted.